cardano-node/src/Cardano/CLI/Tx/Generation.hs

```diff
@@ -187,6 +185,9 @@ genesisBenchmarkRunner loggingLayer
   liftIO . traceWith benchTracer . TraceBenchTxSubDebug
     $ "******* Tx generator, addresses are ready *******"
 
+  -- | Set of available tx outputs (for new transactions).
+  availableFunds :: TVar.TVar AvailableFunds <- liftIO $ TVar.newTVarIO Set.empty
```

Comment:
@denisshevchenko, it doesn't look like `availableFunds` has to be a mutable variable here, I think -- we can use the return value:
```
prepareInitialFunds >>= runBenchmark
```